app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -315,6 +316,10 @@
         };
         return item.tier.toLowerCase() === (tierMap[predicate] || predicate);
       },
+      sublime: function(predicate, item) {
+        return item.isEngram() &&
+          item.name.indexOf('Sublime') === 0;
```

Comment:
Unfortunately, we can't use the name of things for this type of filter, because it won't work in all languages. Instead, we try to look at things like the categories of the item, which are language independent.